connector/src/test/scala/com/microsoft/kusto/spark/KustoWriterTests.scala

```diff
@@ -52,4 +54,31 @@ class KustoWriterTests extends FlatSpec with Matchers {
     verify(buffer, times(1)).close()
   }
 
+  "getColumnsSchema" should "parse table schema correctly" in {
+    //val columns = "{\"Name\":\"Subscriptions\",\"OrderedColumns\":[{\"Name\":\"SubscriptionGuid\",\"Type\":\"System.String\",\"CslType\":\"string\"},{\"Name\":\"Identifier\",\"Type\":\"System.String\",\"CslType\":\"string\"},{\"Name\":\"SomeNumber\",\"Type\":\"System.Int64\",\"CslType\":\"long\"},{\"Name\":\"IsCurrent\",\"Type\":\"System.SByte\",\"CslType\":\"bool\"},{\"Name\":\"LastModifiedOn\",\"Type\":\"System.DateTime\",\"CslType\":\"datetime\"},{\"Name\":\"IntegerValue\",\"Type\":\"System.Int32\",\"CslType\":\"int\"}]}"
```

Comment:
delete?

Reply:
I change a bit but left since this explains the compared schema